btcpay-server/docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       NBXPLORER_BTCHASTXINDEX: 1
 
   web:
-    image: btcpayserver/btcpayserver:1.9.2@sha256:efa1245100cf5ec90e56e0ab216995885074357dce4817d362f2d02fd47c9048
+    image: btcpayserver/btcpayserver:1.11.2@sha256:06177a27d6d5ce5df496a848b1527a14e39606ba5468a00cf26b5b58bd269879
```

Comment:
This can be changed to `1.11.1` now. I get the following digest for 1.11.1:
`44742c74deccb4a42dddf25ff5fce67c3e905eb7f88b047e8f8a80c261bc134f`

Can you please double check?

Reply:
same, updated!